tests/core/pyspec/eth2spec/test/altair/epoch_processing/test_process_sync_committee_updates.py

```diff
@@ -13,20 +17,12 @@
 )
 
 
-@with_all_phases_except([PHASE0])
-@spec_state_test
-@with_configs([MINIMAL], reason="too slow")
-def test_sync_committees_progress(spec, state):
-    current_epoch = spec.get_current_epoch(state)
-    # NOTE: if not in the genesis epoch, period math below needs to be
-    # adjusted relative to the current epoch
-    assert current_epoch == 0
-
+def run_sync_committees_progress_test(spec, state):
     first_sync_committee = state.current_sync_committee
     second_sync_committee = state.next_sync_committee
 
-    slot_at_end_of_current_period = spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD * spec.SLOTS_PER_EPOCH - 1
-    transition_to(spec, state, slot_at_end_of_current_period)
+    end_slot_of_current_period = state.slot + spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD * spec.SLOTS_PER_EPOCH - 1
+    transition_to(spec, state, end_slot_of_current_period)
```

Comment:
this should be working bc `state.slot % spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD == 0` but i think it would break if it was any other value as you would cross over into the next sync committee period while still wanting to be within the current sync committee period.

the simple way is to just leverage our inputs, throw an `assert state.slot % spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD == 0` and call it a day.

i think a better option is to just compute what we are after:

```python
current_period = spec.get_current_epoch(state) // spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD
next_period = current_period + 1
next_period_start_epoch = next_period * spec.EPOCHS_PER_SYNC_COMMITTEE_PERIOD
next_period_start_slot = next_period_start_epoch * spec.SLOTS_PER_EPOCH
end_slot_of_currrent_period = next_period_start_slot - 1
```